app/views/govuk_publishing_components/components/docs/layout_for_public.yml

```diff
@@ -5,6 +5,8 @@ body: |
 
 display_preview: false
 display_html: true
+accessibility_criteria: |
+  TBD
```

Comment:
**Reviewing commit by commit**

TBD

Reply:
I think this was copied over from `/govuk_publishing_components/components/docs/layout_for_admin.yml` which also has "TBD" as its accessibility criteria 😅

This raises a good point though –  should we define accessibility criteria for something that is essentially an HTML wrapper for other components/content?... and if yes, what would that criteria be (assuming that its contents have their own accessibility criteria)

Reply:
We shouldn't define criteria for anything that might be passed to the component, because in this case that would need the whole of WCAG, pretty much. But there might be some specific criteria around setting a `lang` attribute, having a sensible page title, that kind of thing?